defaults/test_default.yaml

```diff
@@ -29,10 +29,14 @@ nemesis_filter_seeds: true
 instance_provision: spot_low_price
 spot_max_price: 0.60
 
-failure_post_behavior: destroy
+execute_post_behavior: false
+post_behavior_db_nodes: keep-on-failure
+post_behavior_loader_nodes: destroy
+post_behavior_monitor_nodes: keep
 
 cloud_credentials_path: '~/.ssh/support'
 
 backtrace_decoding: false
 
 logs_transport: "rsyslog"
+collect_logs: true
```

Comment:
default should be `false`, we we collect logs in Jenkins pipeline (and if somebody wants it in his private test YAML should enable it explicitly)

Reply:
You can't assume some log isn't necessary in analyzing failed jobs. So please save all logs generated by SCT. It's general requirement, not just for private testing.

Reply:
Set to false.
All logs are collected as before. This parameter only disable collect logs in teardown method. 
All logs will be collected by new stage in pipeline.